### DISCUSSION TITLE
Deepgram stability issue

### DISCUSSION THREAD

**Discussion by maketesteasier** [deepgram/community#907] [General help] [answered]
Yesterday I tried to transcribe a couple of mp3 files to text on my website, but I failed. Today, I tried one, it worked very fast. We did not change anything in our end. What's the cause? Any idea? The only reason I can think of is that Deepgram is not stable. Does the fact that I am in calling deepgram api from China affect the stability? How can I find out the real cause behind this?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Reply by maketesteasier** [reply]
Request ID:  15c16a17-f3d1-422d-871a-242a5172fe3b
URL: /v1/listen?model=nova-2&smart_format=true&punctuate=true&diarize=true&utterances=true&paragraphs=true
Endpoint: /listen
Status: 408
Performed at: 08/31/2024 02:31:21
Error: "No information available."

I met this error again. Please help me. I am in urgent need of this function.

**Reply by maketesteasier** [reply]
Also, I tried several requests. The weird thing is that the status was OK on deepgram website, but we did not get the results back. Would you please also check these requests? 

/v1/listen?model=nova-2&smart_format=true&punctuate=true&diarize=true&utterances=true&paragraphs=true
Request details
Request ID
31903347-366a-4418-9362-1d6cc641e059

Endpoint
/listen
Status
200 OK
Method
Synchronous
Performed at
08/31/2024 03:22:17

Another one: 
/v1/listen?model=nova-2&smart_format=true&punctuate=true&diarize=true&utterances=true&paragraphs=true
Request details
Request ID
21dda092-2162-46bc-ba6b-f601a49507c1
Endpoint
/listen
Status
200 OK
Method
Synchronous
Performed at
08/31/2024 03:11:20

**Reply by lukeocodes** [reply, marked as answer]
Given that you got a [408](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/408), I am going to go ahead and assume it was a connectivity issue.

There is a de-peering problem between an ISP in the US and Tata Telecommunications, which has led to extreme instability from the APAC region. If you could try a VPN to the US first, you may find better stability

https://deepgram.gitbook.io/help-center/notices/cogent-tata-peering-issue-impact-on-deepgram

**Reply by maketesteasier** [reply]
Hi, thanks for the reply. Currently, I am using cloud service from Alibaba, whose servers are supposed to be in Beijing or somewhere in China. I do not know if I can get better stability if I switch to a server located in Singapore.
